.gitignore

```diff
@@ -14,3 +14,4 @@ output/
 markdownissues.txt
 node_modules
 package-lock.json
+secret-local.ps1
```

Comment:
Usually I'd rather use secret[s].local.ps1 so that the `*.local.*` entry picks it up.